client/my-sites/migrate/section-migrate.scss

```diff
@@ -89,6 +89,7 @@
 	display: block;
 	margin: auto;
 	width: 260px;
+	height: 202px;
```

Comment:
This seems like a "magic number" -- why 202px? If something with the layout changes, say text when translated, would this number no longer work?

Reply:
202.367px was the "natural" height of the illustration in my browser, so I set it at that, but 200px works equally well and may puzzle people less. Adding four times as much text to the paragraph below the illustration doesn't change the layout, and it behaves well for me at different viewport sizes.